### DISCUSSION TITLE
Telescope Control Plugin not showing

### DISCUSSION THREAD

**Discussion by foxmike80** [Stellarium/stellarium#4028] [General]
Hello, I'm a new user trying to setup my nexstar telescope with stellarium 24.4 on Linux. In the plugins sections I do not see the entry "telescope control". What am I missing?

**Comment by blue-alien-bg** [comment]
Hello @foxmike80, did you find a solution. I'm in a similar situation.

**Reply by 10110111** [reply]
How did you install Stellarium?

**Reply by blue-alien-bg** [reply]
I installed it on an Arch Linux system using Discover. In particular a Steam Deck because I don't have any other portable PC.

**Reply by 10110111** [reply]
Is it a vanilla Arch Linux or SteamOS? If the latter, I can imagine that the packagers there decided to exclude the plugin from the distribution (would be a strange decision, since Steam Deck has a USB-C port, if I'm not mistaken). You can try an AppImage version instead.

**Reply by blue-alien-bg** [reply]
Sorry, it IS SteamOS not vanilla Arch.
I didn't know about AppImage-s, I'll look into it.
Isn't there a way to add the plugin like an external one?

**Reply by 10110111** [reply]
> Isn't there a way to add the plugin like an external one?

Well, you'd need to take it from somewhere, compiled in a compatible way with your Stellarium binary. It would be easier to just use the whole package that includes this plugin. Maybe from the official Arch Linux repos (if it's easy enough to do), or, failing that, the AppImage.

**Reply by blue-alien-bg** [reply]
Thank you for the suggestions! I'll first try the AppImage and see if that works for my case.

**Comment by mimigernaford** [comment]
Hello everyone,

while installing Stellarium 25.4 on a Linux PC via Flathub, I noticed that the telescope plugin is missing from the settings menu.
Version 20.4 in the Mint system package still includes it.
Are there any alternatives?

Regards,
Michael

**Reply by DeltaZulu99** [reply]
I have the same issue on 25.3 on Fedora 43. No Telescope control plugin. Fresh install of Fedora and Stellarium. I tried installing Stellarium via Discover and dnf cli. Same result.

**Reply by DeltaZulu99** [reply]
Using the 25.4 appimage solved the issue.
